sound/sound_banks/0C_extended_bank.json

```diff
@@ -472,37 +514,74 @@
             "envelope": "envelope30",
             "sound": "18_timpani"
         },
-        "Drump Sample 26": {
+        "Drum Sample 26": {
             "release_rate": 10,
             "normal_range_lo": 18,
             "envelope": "envelope31",
             "sound_lo": "12_snare_drum_1",
             "sound": "13_snare_drum_2"
         },
-        "Drump Sample 27": {
+        "Drum Sample 27": {
             "release_rate": 10,
             "normal_range_lo": 18,
             "envelope": "envelope32",
             "sound_lo": "12_snare_drum_1",
             "sound": "13_snare_drum_2"
         },
-        "Xylophone": {
+        "Music Box": {
             "release_rate": 119,
             "envelope": "envelope28",
             "sound": "3A_music_box"
         },
-        "Church Organ": {
+        "Synth Voice 2": {
             "release_rate": 19,
             "normal_range_hi": 50,
             "envelope": "envelope0",
             "sound": {"sample": "1B_organ_2", "tuning": 1.0},
             "sound_hi": "1B_organ_2"
         },
-        "Bells": {
+        "Sleigh Bells 2": {
             "release_rate": 208,
             "envelope": "envelope33",
             "sound": "2F_sleigh_bells"
         },
+        "Strings Ensemble": {
+            "release_rate": 24,
+            "normal_range_lo": 42,
+            "envelope": "envelope34",
+            "sound_lo": "14_strings_5",
+            "sound": "15_strings_4"
+        },
+        "Trumpet": {
+            "release_rate": 67,
+            "envelope": "envelope35",
+            "sound": "17_trumpet"
+        },
+        "Harpsichord": {
+            "release_rate": 255,
+            "envelope": "envelope36",
+            "sound": "27_harpsichord"
+        },
+        "Monk Choir": {
+            "release_rate": 10,
+            "envelope": "envelope37",
+            "sound": "3E_monk_choir"
+        },
+        "Low Cowbell": {
+            "release_rate": 10,
+            "envelope": "envelope38",
+            "sound": "3F"
+        },
+        "High Cowbell": {
+            "release_rate": 10,
+            "envelope": "envelope39",
+            "sound": "40_bell"
+        },
```

Comment:
Not sure I could classify these as cowbells, though I don't really know what they actually are

Reply:
i'll rename them to just "bell"